### DISCUSSION TITLE
How to build GBM?

### DISCUSSION THREAD

**Discussion by Darthagnon** [MikeMaximus/gbm#210] [Q&A] [answered]
I was trying to build GBM the other day from source, as I really wanted to test out the new layouts, but Visual Studio complained about a missing resource file. Are there some special steps I need to take to build it? I'm attempting to do so via Visual Studio 2017 on Windows 10 v1709.

**Comment by MikeMaximus** [comment, marked as answer]
There shouldn't be anything special required.  Do you have a more exact error message?  Is it complaining about BuildDate.txt?

**Reply by Darthagnon** [reply]
As Murphy's Law would have it, I tried building again off some recent commits, and it built just fine without any errors. 

Possibly I was trying to build, starting from the wrong file last time. I can't remember the exact error code (that I haven't seen since) but it was something like "can't build due to missing Game.Backup.Monitor.resource". 

I now have working beta binaries for v1.25; am comparing the tabbed build with the non-tabbed build.
(noticed a small alignment bug: the games list in Launcher mode goes out of alignment by 5px or so at the bottom when GBM is fullscreened)

**Reply by MikeMaximus** [reply]
Not sure what was causing that, but glad it's working for you now.  

Yeah I noticed that as well with the list, it's an issue with the listbox control itself.  When it grows or shrinks, it will only grow in certain increments, instead 1px at a time.  So as soon as you resize the window it starts going a bit crazy.  I'll have to play with the default sizes, maybe I can get it to stop doing that if I make it a specific size and align it.

[EDIT] The alignment issue is fixed in 83ab3be.
